README.tfdoc.hcl

```diff
@@ -426,8 +425,8 @@ section {
       }
 
       variable "traffic" {
-        type        = list(traffic)
-        default     = []
+        type        = any
+        default     = null
```

Comment:
please remove `default = null` from every tfdoc you see. btw why did you change this from list to `any`?

Reply:
I only updated it so it would reflect `variables.tf` but on second thought, it makes more sense to have the documentation state `list(traffic)`. I will remove `default = null` from the `README.tfdoc.hcl`. Thanks